--- tensorflow-for-lima-1.9.orig/tensorflow/core/common_runtime/local_device.h
+++ tensorflow-for-lima-1.9/tensorflow/core/common_runtime/local_device.h
@@ -37,13 +37,13 @@ class LocalDevice : public Device {
               const DeviceAttributes& attributes);
   ~LocalDevice() override;
 
- private:
-  static bool use_global_threadpool_;
-
   static void set_use_global_threadpool(bool use_global_threadpool) {
     use_global_threadpool_ = use_global_threadpool;
   }
 
+ private:
+  static bool use_global_threadpool_;
+
   struct EigenThreadPoolInfo;
   std::unique_ptr<EigenThreadPoolInfo> owned_tp_info_;
 
